src/components/paragraph.tsx

```diff
@@ -0,0 +1,28 @@
+// ----- Imports ----- //
+
+import { FC, ReactNode } from 'react';
+import React, { css } from '@emotion/core';
+import { body } from '@guardian/src-foundations/typography';
+
+import { spaceToRem } from 'styles';
+
+
+// ----- Component ----- //
+
+interface Props {
+    children?: ReactNode;
+}
+
+const styles = css`
+    ${body.medium()}
+    overflow-wrap: break-word;
+    margin: 0 0 ${spaceToRem(3)};
```

Comment:
can this not now be `remSpace`? (referring to your other PR)

Reply:
👍 got things a bit muddled, should be updated now.